### PR TITLE
Additions in protocol.txt PAYBACKLOAN

### PR DESCRIPTION
**Pull request by Rubberduck1**
Please can you merge that pull request into main. Thanks @Jabul99

### CODE REVIEW

File: protocol.txt [hunk 1]
```diff
@@ -35,5 +35,9 @@ ACCEPTLOAN <Loan Number> <Account>
 e.g. ACCEPTLOAN 1 11111111
 Returns: Success. Loan number 1 accepted by account 11111111.
```
Comment (Jabul99) [inline protocol.txt:36]: Ah great, that you thought of this change. While we're at it, could you also update the HELP command?

Comment (Jabul99) [reply protocol.txt]: nicely done

**Comment by Rubberduck1** [comment]
[HELP Command updated](https://trello.com/c/YXlpoBTc/45-help-command-updated)

File: protocol.txt (hunk 1 above)
Comment (Rubberduck1) [reply protocol.txt]: Thanks @Jabul99

**Pull request merged**